### PR TITLE
add status command for crictl

### PR DESCRIPTION
**Pull request by heartlock**
Signed-off-by: heartlock <21521209@zju.edu.cn>
ref #33 
@feiskyer @xlgao-zju

### CODE REVIEW

File: cmd/crictl/status.go [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+	},
+}
+
+// Status sends a StatusRequest to the server, and parses the returned StatusResponse.
+func Status(client pb.RuntimeServiceClient) error {
+	r, err := client.Status(context.Background(), &pb.StatusRequest{})
+	if err != nil {
+		return err
+	}
+	fmt.Println("StatusResponse:")
```
Comment (feiskyer) [inline cmd/crictl/status.go]: replace with `Runtime contitions: `

File: cmd/crictl/status.go [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+	var runtimeCondition *pb.RuntimeCondition
+	for _, runtimeCondition = range r.GetStatus().GetConditions() {
```
Comment (feiskyer) [inline cmd/crictl/status.go]: Add an empty line for each condition.

File: cmd/crictl/status.go [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+	"github.com/urfave/cli"
+	"golang.org/x/net/context"
+	pb "k8s.io/kubernetes/pkg/kubelet/api/v1alpha1/runtime"
+)
+
+var runtimeStatusCommand = cli.Command{
+	Name:  "status",
+	Usage: "get the status of runtime",
+	Action: func(context *cli.Context) error {
+		// Set up a connection to the server.
```
Comment (xlgao-zju) [inline cmd/crictl/status.go]: I think we can extract the logic of `Set up a connection to the server` as a util function. And put it in util.go.

Comment (xlgao-zju) [inline cmd/crictl/status.go]: @feiskyer WDYT?

Comment (heartlock) [inline cmd/crictl/status.go]: I think it is overdesign here

Comment (feiskyer) [inline cmd/crictl/status.go]: Yep, I think so.

@xlgao-zju Does the urfave/cli support global defer action? or else, we need to close the connection in each subcommand.

Comment (xlgao-zju) [inline cmd/crictl/status.go]: @feiskyer @heartlock 

We have `After` see [urfave/cli/#151](https://github.com/urfave/cli/pull/151) . And we also have companion action of `After` which called `Before`.

Comment (feiskyer) [inline cmd/crictl/status.go]: Cool. @xlgao-zju Could you make extraction after this PR?

Comment (xlgao-zju) [inline cmd/crictl/status.go]: @feiskyer You mean that I can file a new PR, after this PR is merged? Or file a new PR now?

Comment (xlgao-zju) [inline cmd/crictl/status.go]: File #52 to fix this.
cc @feiskyer @heartlock

**Comment by feiskyer** [comment]
@heartlock Could update the PR by using same connection func introduced in #52

**Comment by heartlock** [comment]
sure, I will update it.

**Comment by heartlock** [comment]
updated it ! @feiskyer

**Comment by feiskyer** [comment]
Thanks. LGTM.

**Pull request merged**